certificate.go

```diff
@@ -94,17 +94,21 @@ func (c Certificate) Expires() time.Time {
 // GetFingerprints returns the list of certificate fingerprints, one of which
 // is computed with the digest algorithm used in the certificate signature.
 func (c Certificate) GetFingerprints() ([]DTLSFingerprint, error) {
-	fingerprintAlgorithms := []dtls.HashAlgorithm{dtls.HashAlgorithmSHA256}
+	fingerprintAlgorithms := []string{"sha-256"}
```

Comment:
This not being an enum anymore is kind of unfortunate :/ I could see mistakes coming from this.

Reply:
But then exporting bloats the public API. I don't know which tradeoff is worse.

Reply:
We could add inverse func of https://github.com/pion/dtls/blob/master/pkg/crypto/fingerprint/hash.go. Having it would be good.

Reply:
https://github.com/pion/dtls/pull/186

Reply:
Change looks fantastic. I left my comments on other PR, but all approves from me :)